Hummingbird.docc/Articles/JSON.md

```diff
@@ -0,0 +1,21 @@
+# JSON Support
+
```

Comment:
Do we need a separate page just for JSON?
Perhaps we can change the byline for the `Encoding and Decoding` article to make it clearer that it is the how you decode JSON data.

Reply:
eg "Support for decoding and encoding JSON and other file formats with Codable"

Reply:
For a newcomer to Hummingbird 2 _and_ Swift, I feel that En/Decoding doesn't really tag onto what you want to do. People want to send/receive JSON, they "don't care" that this requires Codable.

Reply:
I would prefer we fix the naming of encoding/decoding docs instead of repeating what is pretty much the same content in this JSON doc.